tests/Aspire.EndToEnd.Tests/IntegrationServicesFixture.cs

```diff
@@ -269,8 +275,22 @@ public async Task DumpDockerInfoAsync(ITestOutputHelper? testOutputArg = null)
         testOutput.WriteLine("--------------------------- Docker info (end) ---------------------------");
     }
 
-    public async Task DumpComponentLogsAsync(string component, ITestOutputHelper? testOutputArg = null)
+    public async Task DumpComponentLogsAsync(TestResourceNames resource, ITestOutputHelper? testOutputArg = null)
     {
+        string component = resource switch
```

Comment:
(not for this PR, but we can fix it in main) - 

This is kind of weird now that `TestResourceNames` is a flags enum. If you pass in a value with multiple resources set, it will throw with "unknown resource: {resource}". Maybe the error message could be better saying only single resource names are supported?